Rakefile

```diff
@@ -1 +1,11 @@
 require 'bundler/gem_tasks'
+
+begin
+  require 'rspec/core/rake_task'
+
+  RSpec::Core::RakeTask.new(:spec)
+
+  task :default => :spec
+rescue LoadError
+  # no rspec available
+end
```

Comment:
I think we can remove this `rescue` because we have the rspec dependency listed in the gemspec. If a developer forgets to use `bundle exec` that is a legitimate mistake and they should see an exception.